src/components/LoginForm.jsx

```diff
@@ -23,11 +25,12 @@ const LoginForm = (props) => {
         history.goBack();
       }
     } catch (error) {
-      setMessage(error.response.data.errors[0]);
+      seterrorMessage(error.response.data.errors[0]);
```

Comment:
This should be camelCased to setErrorMessage actually, as well as when declared on line 11